api/src/main/java/io/grpc/ServerCallHandler.java

```diff
@@ -25,17 +25,26 @@
 @ThreadSafe
 public interface ServerCallHandler<RequestT, ResponseT> {
   /**
-   * Produce a non-{@code null} listener for the incoming call. Implementations are free to call
-   * methods on {@code call} before this method has returned.
+   * Passes {@code call} on to the next stage of asynchronous processing.
```

Comment:
"next stage" is weird terminology for this interface. This interface exists separate from interceptors and is more foundational than interceptors. How about something like "Start processing a new incoming call."?

Reply:
done